### ISSUE TITLE
SENS_EN_PX4FLOW not working on fmuv5 (Pixhawk4)

### ISSUE THREAD

**Issue by ucwxb** [PX4/PX4-Autopilot#18404]
### Describe the bug
If set SENS_EN_PX4FLOW to 1, PX4FLOW still not start on fmuv5
need to run px4flow start -X everytime

px4 version: 1.12.3

### Log Files and Screenshot
![113808167-046ce280-9798-11eb-8d0a-f0134ba5a69d](https://user-images.githubusercontent.com/29229101/136764172-12f0a3df-cc74-41a3-a165-2d137748ff1b.png)
![113808190-1189d180-9798-11eb-9631-7cf0630050e5](https://user-images.githubusercontent.com/29229101/136764194-c1012f69-8589-4c5a-9b00-4068a198e4b4.png)
s

**Comment by bresch** [comment]
Is the FMU booting faster than the px4 flow maybe? Do you have the same issue if you `reboot` from the console instead of power-cycling the setup?

**Comment by ucwxb** [comment]
> Is the FMU booting faster than the px4 flow maybe? Do you have the same issue if you `reboot` from the console instead of power-cycling the setup?

Yes, that's the same when I reboot the pixhawk4. Finally we solved it by using the MAVLINK message "Serial Control" sent by stm32,which could send command "px4flow start -X\n". This method works.

**Comment by bresch** [comment]
@ucwxb maybe try a `dmesg` in the console after boot to check if the autopilot reports an error while starting px4flow.

**Comment by hamishwillee** [comment]
Closing. No response in 4 years.
